### DISCUSSION TITLE
Mac OS X: Top margin on top window buttons. How to set?

### DISCUSSION THREAD

**Discussion by timahrentlov** [AvaloniaUI/Avalonia#13386] [Q&A]
I'm experimenting on a Mac/Sonoma for how to build a main window that looks and performs sort of like the Weather App. Right now, I'm just trying to create some space above the close/minimize/restore buttons in the top left corner.

If you look at this image, you'll notice that the weather app (to the left) have more space than my genius app (to the right).  

<img width="500" alt="Screenshot 2023-10-25 at 14 46 33" src="https://github.com/AvaloniaUI/Avalonia/assets/1124919/b36c51ba-18ca-48b2-955c-139fa782a83f">

Anyone knows how I can set this? Tried styles, poking values into the TitleBar in the code behind, and so on. 
It just seems totally indifferent to my feeble attempts.

**Comment by kaiytech** [comment]
After spending more time on it than I'd like to admit, finally, found it!

If you add both of these flags, the buttons will be lower :)

```
ExtendClientAreaChromeHints |= ExtendClientAreaChromeHints.NoChrome | ExtendClientAreaChromeHints.OSXThickTitleBar;
```
